### PR TITLE
PR -- basic layout to release 0.0.1

### PR DESCRIPTION
**Pull request by jeremyrist**

### CODE REVIEW

**Comment by boshnivolo** [comment]
Looking good! :+1:

**Comment by karjac14** [comment]
Great layout there!

File: src/index.html [hunk 1]
```diff
@@ -15,12 +15,11 @@
         <script src="js/vendor/modernizr-2.8.3.min.js"></script>
     </head>
     <body>
-        <!--[if lt IE 8]>
-            <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
-        <![endif]-->
+      <div class="container">
```
Comment (kellymurray) [inline src/index.html:18]: How can you make this more semantic?

File: src/scss/_custom.scss [hunk 2]
```diff
@@ -1,3 +1,29 @@
-p {
-  background-color: blue;
+* {
+  box-sizing: border-box;
+}
+body {
```
Comment (kellymurray) [inline src/scss/_custom.scss:4]: Could you could make this more efficient? Why might you not want to nest them all under the `body`?

**Pull request merged**